.github/workflows/CI.yaml

```diff
@@ -138,6 +138,8 @@ jobs:
           cache: gradle
       - name: Build framework '${{ matrix.framework }}'
         run: ./gradlew dataland-framework-toolbox:runCoverage --args='${{ matrix.framework }}' --no-daemon --stacktrace
+        env:
+          DATALAND_QA_SERVICE_CLIENT_SECRET: "dummy"
```

Comment:
Wieso braucht man in Toolbox Tests das client-secret?

Reply:
Da der QA Service für die OpenAPi-Generierung gestartet wird.